flink-sliding-feature-view-benchmark/run_and_verify.sh

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+cd "$(dirname "$0")"
+PROJECT_DIR=$(cd "$(pwd)/.."; pwd)
+source "${PROJECT_DIR}"/tools/utils.sh
+
+docker-compose up -d
+wait_for_port 8081 "Flink Cluster"
+
+python main.py --records-num 1000
+
```

Comment:
Do we need `exit_code=$?` here similar to `flink-read-write-redis/run_and_verify.sh`?

If we don't need it, can you double check that nightly-build workflow will report error if the main.py does not finish successfully?

Reply:
I don't think we need `exit_code=%?` as we `set -e` at the beginning of the script. 

I test it in this github action [run](https://github.com/flink-extended/feathub-examples/actions/runs/3965322758/jobs/6794945758).

Reply:
Sounds good.